server/cmd/mmctl/commands/import.go

```diff
@@ -251,11 +251,11 @@ func importProcessCmdF(c client.Client, command *cobra.Command, args []string) e
 	importFile := args[0]
 
 	isLocal, _ := command.Flags().GetBool("local")
-	isLegacy, _ := command.Flags().GetBool("legacy")
+	bypassUpload, _ := command.Flags().GetBool("bypass-upload")
 	// in local mode, we tell the server to directly read from this file.
-	if isLocal && !isLegacy {
+	if isLocal && bypassUpload {
 		if _, err := os.Stat(importFile); errors.Is(err, os.ErrNotExist) {
-			return fmt.Errorf("file %s doesn't exist. NOTE: If this file was uploaded to the server, please use the --legacy flag to revert to old behavior.", importFile)
+			return fmt.Errorf("file %s doesn't exist. NOTE: If this file was uploaded to the server via mmctl import upload, please use the --legacy flag to revert to old behavior.", importFile)
```

Comment:
This error mentions the `--legacy` flag, but that no longer exists

```suggestion
			return fmt.Errorf("file %s doesn't exist. NOTE: If this file was uploaded to the server via mmctl import upload, please omit the --bypass-upload flag to revert to old behavior.", importFile)
```